src/model/ScheduleTypeRegistry.cpp

```diff
@@ -94,6 +94,14 @@ namespace model {
        1.0},
       {"AirConditionerVariableRefrigerantFlow", "Thermostat Priority Schedule", "thermostatPrioritySchedule", false, "ControlMode", 0.0, 1.0},
       {"AirConditionerVariableRefrigerantFlow", "Basin Heater Operating Schedule", "basinHeaterOperatingSchedule", false, "Availability", 0.0, 1.0},
+      {"AirflowNetworkOccupantVentilationControl", "Opening Probability", "openingProbabilitySchedule", true, "", 0.0, 1.0},
+      {"AirflowNetworkOccupantVentilationControl", "Closing Probability", "closingProbabilitySchedule", true, "", 0.0, 1.0},
```

Comment:
Note for opening/closing prob, it's expected in [0, 1]

cf https://github.com/NREL/EnergyPlus/blob/acda84e38db4e8aecbc178cb970055d3721ee321/src/EnergyPlus/AirflowNetwork/src/Solver.cpp#L12908-L12914